sql/timeseries/pwaScores.sql

```diff
@@ -0,0 +1,44 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION
```

Comment:
I find BigQuery's "format query" function generates some pretty ugly formatting. Could you copy the style from this [crux query](https://github.com/HTTPArchive/bigquery/blob/master/sql/histograms/cruxFcp.sql) with a UDF?

Another thing to reformat, keep the fields on one line, eg

```sql
ROUND(APPROX_QUANTILES(lighthouse_pwa_score, 101)[OFFSET(11)], 2) AS p10,
```

Reply:
I fully agree about the ugly formatting, just thought we settled for whatever it produces. Have changed to a more sane style manually now. (FYI, I have opened an issue with the formatter internally.)